app/src/main/java/nl/tudelft/b_b_w/model/HashException.java

```diff
@@ -3,5 +3,5 @@
 /**
  * Thrown when the hash methods are not available.
  */
-public class HashException extends Exception {
+public class HashException extends RuntimeException {
```

Comment:
Hmmm, valid point. Although crashing apps isn't something you want, even when the app is useless.